src/__mocks__/course.ts

```diff
@@ -1,12 +1,15 @@
-export const mockCourseResponse = {
+import { Course } from '@src/generic/course-card/types';
+
+export const mockCourseResponse: Course = {
   id: 'course-v1:edX+DemoX+Demo_Course',
   data: {
     id: 'course-v1:edX+DemoX+Demo_Course',
     course: 'Demo Course',
     start: '2024-04-01T00:00:00Z',
     imageUrl: '/asset-v1:edX+DemoX+Demo_Course+type@asset+block@course_image.jpg',
     org: 'edX',
-    orgImg: '/asset-v1:edX+DemoX+Demo_Course+type@asset+block@org_image.jpg',
+    orgImageUrl: '/asset-v1:edX+DemoX+Demo_Course+type@asset+block@org_image.jpg',
```

Comment:
[inform]: Changing `orgImg` to `orgImageUrl` is in preparation for a fix in future backend PRs